src/graphql/query/myKiva.graphql

```diff
@@ -35,7 +35,7 @@ query myKivaQuery {
       filter: {
         category: loan
       }
-      limit: 100
+      limit: 15
```

Comment:
As transactions query is also returning `loan_repayment` cases, are we guaranteed that we'll show at least 9 active loans in the carousel?

Reply:
The problem is that it sounds like the transactions query is fairly expensive for the monolith. So we can try 15 here and see how it works for a few users. Sound good?

Reply:
Probably makes sense, what about default limit of 20?
Issue I see is that an user who is not lending but receiving repayments, this query could start hiding loans in the carousel at some point.
Also, should be useful to know what is the default value for `endDate` filter.
Example: old loans query limit was 50; for Shua's fake auth case, having a limit of 100 here, we were able to get 36 loans after filtering. As showing 9 is not really needed, but useful to ensure we are able to show 9 at least.